pkg/auth/dependency/webapp/render_provider_impl.go

```diff
@@ -54,8 +53,11 @@ func (p *RenderProviderImpl) WritePage(w http.ResponseWriter, r *http.Request, t
 		return MakeURLWithPath(r.URL, path)
 	}
 
-	data["client_name"] = accessKey.Client["client_name"]
-	data["logo_uri"] = accessKey.Client["logo_uri"]
+	preferredLanguageTags := intl.GetPreferredLanguageTags(r.Context())
+
+	clientMetadata := accessKey.Client
+	data["client_name"] = intl.LocalizeOIDCJSONObject(preferredLanguageTags, clientMetadata, "client_name")
```

Comment:
It seems currently there is no way to trigger this, as access key is read from header only?

Reply:
Auth UI now understand `redirect_uri`, `client_id` and `ui_locales`.